src/sort.js

```diff
@@ -11,54 +11,62 @@ const sorter = function(direction, a, b) {
   return direction;
 };
 
+const compareSorter = function(comparer) {
```

Comment:
nitpicking: Thinking it would be bit more clear if we name this function `customComparerProvider` and also rename above `sorter` function to `defaultComparer` wdyt?

Reply:
Renamed